app/stores/UsersStore.js

```diff
@@ -77,6 +77,7 @@ export default class UsersStore extends BaseStore<User> {
     runInAction(`invite`, () => {
       res.data.users.forEach(this.add);
     });
+    this.total += res.data.sent.length;
```

Comment:
I think this is the right way of handling the total count since it's an observable?

Reply:
Yep, this is fine – would be slightly improved by being inside the `runInAction`